include/sdsl/int_vector.hpp

```diff
@@ -1192,7 +1432,21 @@ int_vector<t_width>::~int_vector()
 template <uint8_t t_width>
 void int_vector<t_width>::bit_resize(const size_type size)
 {
-	memory_manager::resize(*this, size);
+	assert(size % m_width == 0);
```

Comment:
It would be nice if the sdsl::int_vector would behave like the std::vector or folly:fbvector. I'll go through the code this weekend and check where the vector actually shrinks when resize is called and see whether we can add a call to ``shrink_to_fit()`` there.

Reply:
I went through all the code and added `shrink_to_fit()` where the int_vector seems to be expected to shrink.